### PR TITLE
Improved Unet with ISIC 2018 dataset

### PR DESCRIPTION
**Pull request by JosephAaron3**
This pull includes a single directory in /recognition containing: a model script with a self-contained Improved Unet Keras model, a driver script to run the file, a readme for information about the project, a tb directory for Tensorboard output, and an image directory for various outputs and descriptive images of the algorithm.

Joseph Menesch, s4393986

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages (some, but not many, could be improved)
- Code valid for Normal Difficulty
- Results are good

Other Feedback
- Good
- (OPTIONAL) I would break up long functions into reusable chunks when building the model. It makes hyper parameter adjustment much easier.
- (OPTIONAL) Would put main logic in `if __name__ == '__main__'` for the sake of following python convention.

**Comment by rexsw** [comment]
Code looks good for normal difficulty

For next time;

Comments such as 
   #Print data size info and sample
   print("\nData info:")
Add nothing to the readability of the code and should be avoided. But overall comments are good. 

Code in files such as driver.py should be in a main function.

File: recognition/ISIC_segmentation_43939862/driver.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
+import glob
```
Comment (shakes76) [inline recognition/ISIC_segmentation_43939862/driver.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. remove unnecessary files like the event files.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by JosephAaron3** [comment]
> **Feedback required. remove unnecessary files like the event files.**

Done

**Pull request merged**